### ISSUE TITLE
Does not install with python3.11.

### ISSUE THREAD

**Issue by vivaansinghvi07** [PaddlePaddle/Paddle#55050]
### 问题描述 Issue Description

Running `pip install paddlepaddle` gives the following error: 

ERROR: No matching distribution found for paddlepaddle==2.4.2`

```
ERROR: Could not find a version that satisfies the requirement paddlepaddle (from versions: none)
ERROR: No matching distribution found for paddlepaddle
```

### 版本&环境信息 Version & Environment Information

OS: Mac
Python: 3.11
Pip: 23.0.1

**Comment by JZ-LIANG** [comment]
python -m pip install paddlepaddle==2.4.2 -i https://pypi.tuna.tsinghua.edu.cn/simple

**Comment by JZ-LIANG** [comment]
https://www.paddlepaddle.org.cn/install/old?docurl=/documentation/docs/zh/install/pip/macos-pip.html
更多 mac 版本请参考

**Comment by vivaansinghvi07** [comment]
Neither works. For searching in indexes, it says that it cannot find it. I fixed it by downgrading to python 3.9, so I'll close the issue.
